Source/LinuxParking.API/Database/Repositories/ParkingRepository.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LinuxParking.API.Database.Context;
+using LinuxParking.API.Domain.Interfaces.Repositories;
+using LinuxParking.API.Domain.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace LinuxParking.API.Database.Repositories
+{
+    public class ParkingRepository : BaseRepository, IParkingRepository
+    {
+        public ParkingRepository(AppDbContext ctx) : base(ctx) { }
+
+
+        public async Task AddAsync(ParkingStatus parking)
+        {
+            await _ctx.ParkingStatuses.AddAsync(parking);
+        }
+
+        public void Delete(ParkingStatus parking)
+        {
+            _ctx.ParkingStatuses.Remove(parking);
+        }
+
+        public async Task<ParkingStatus> FindByIdAsync(int stationId, int parkingId)
+        {
+            return await _ctx.ParkingStatuses.FirstOrDefaultAsync(parking => parking.StationId == stationId && parking.Id == parkingId);
+        }
+
+        public async Task<IEnumerable<ParkingStatus>> ListAsync(int stationId)
+        {
+            return await _ctx.ParkingStatuses.Where(parking => parking.StationId == stationId).ToListAsync();
+        }
+
+        public void Update(ParkingStatus parking)
```

Comment:
I would prefer this method (and also Delete) to be async with a return Ok() to verify success 👍

Reply:
I would also prefer them to be async, but EF dosen't have `DeleteAsync()` nor `UpdateAsync()`.
With `Ok()`, do you mean [ApiController.Ok](https://docs.microsoft.com/en-us/dotnet/api/system.web.http.apicontroller.ok?view=aspnetcore-2.2)?